transform-plugins/src/test/java/co/cask/hydrator/plugin/RunConfigTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.hydrator.plugin;
+
+import co.cask.cdap.api.data.schema.Schema;
+import co.cask.cdap.etl.mock.common.MockPipelineConfigurer;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Test cases for RunConfig.
+ */
+public class RunConfigTest extends TransformPluginsTestBase {
```

Comment:
In earlier PR's, we had suggestions to create the separate test class related to configuration. Hence, written separate test classes.
Should we combine in this one test class? Please let me know.